docs/usage/webhooks.md

```diff
@@ -7,10 +7,42 @@ If using in the Rails framework, we highly recommend you use the [shopify_app](h
 
 ## Create a Webhook Handler
 
+### New webhook handler as of Version 13.5.0
```

Comment:
This is a bit of a nit, but I think we can just make this the "right" way of doing it in this doc, and then just add a quick note with an example on how it used to be. WDYT?